src/providers/completion/styleCompletionItemProvider.ts

```diff
@@ -107,7 +107,7 @@ export class StyleCompletionItemProvider implements CompletionItemProvider {
 					label: key,
 					kind: CompletionItemKind.Class,
 					detail: value.apiName,
-					insertText: new SnippetString(`${key}": {\n\t\${1}\t\n}`)
+					insertText: key
```

Comment:
Is there a way we can keep this snippetstring behaviour where it takes a user down to the next line?